docs/user-documentation/add-member-to-collection.md

```diff
@@ -0,0 +1,13 @@
+# Add a Member to a Collection
+
+To add a member to a collection, please follow these steps:
+
+1. Click the '+Add Content' button
+2. Select 'Repository Item'
+3. Complete the form. At a minimum, an item requires a title and a Content Model type at System > Model (e.g.`Image')
+4. To associate an item with a Collection, please start typing the name of the Collection in the `Member Of' field. The Collection should appear in a dropdown list which can then be selected. 
+5. Click the 'Save' button at the end of the form
+
+You will now see the Drupal node of the item you just created. You will see a field 'Member of' which links to the Collection node which now contains this item. If you click on the name of the Collection you will view the Collection node and see the item listed under `Members'.
+
+Please note that this item does not yet contain any media. Please proceed to [Make Media (and upload a file)](https://islandora-claw.github.io/CLAW/)
```

Comment:
The path for this link will be docs/user-documentation/loading-media.md ( once the pull request for that new page is merged)